base_layer/wallet/migrations/2019-06-26-130555_initial/up.sql

```diff
@@ -1,29 +1,25 @@
-CREATE TABLE sent_messages (
-   id TEXT PRIMARY KEY NOT NULL,
-   source_pub_key TEXT NOT NULL,
-   dest_pub_key TEXT NOT NULL,
-   message TEXT  NOT NULL,
-   timestamp DATETIME NOT NULL,
-   acknowledged INTEGER NOT NULL DEFAULT 0,
-   is_read INTEGER NOT NULL DEFAULT 0,
-   FOREIGN KEY(dest_pub_key) REFERENCES contacts(pub_key)
-);
+-- CREATE TABLE sent_messages (
```

Comment:
Shouldn't you just nuke these rather than comment them out, since you delete the diesel table definitions in `schema.rs`?

Reply:
schema.rs is generated from these migrations and these will come back in if we do text messaging so I thought best to leave them here for the future.